### PR TITLE
Updating the playbook with application-credential issue

### PR DESCRIPTION
**Pull request by raj-prince**
>> Adding the reference of playbook in the main README doc.
>> Updating the playbook with application-credential issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,3 +191,6 @@ Versioning][semver]. Note that the current major version is `0`, which means
 that we reserve the right to make backwards-incompatible changes.
 
 [semver]: http://semver.org/
+
+# GCSFuse common errors/issues
+Please refer [playbook.md](docs/playbook.md), before reporting any issue.
```
Comment (vadlakondaswetha) [inline README.md:196]: do you want to add this line under Support section.

File: docs/playbook.md [hunk 2]
```diff
@@ -36,6 +36,14 @@ Command to enable the flag:
 
 Refer to know more about CGO: https://www.golinuxcloud.com/cgo-tutorial/
 
+### Mount gets stuck with error: DefaultTokenSource: google: could not find default credentials.
+This state comes when we don't provide key-file as a mounting argument. In this
+case GCSFuse tries to fetch the default application credential, but got stuck
+due to absense of application-credentials on the VM.
+
+Resolve this issue by fetching the creds with the below command:  
+**gcloud auth application-default login**
```
Comment (vadlakondaswetha) [inline docs/playbook.md:45]: why are we not following single format in the doc. in the previous cl, you added command under quotes now you made it bold. Please follow single format throughout the doc

File: docs/playbook.md [hunk 3]
```diff
@@ -36,6 +36,14 @@ Command to enable the flag:
 
 Refer to know more about CGO: https://www.golinuxcloud.com/cgo-tutorial/
 
+### Mount gets stuck with error: DefaultTokenSource: google: could not find default credentials.
+This state comes when we don't provide key-file as a mounting argument. In this
+case GCSFuse tries to fetch the default application credential, but got stuck
+due to absense of application-credentials on the VM.
+
+Resolve this issue by fetching the creds with the below command:
```
Comment (vadlakondaswetha) [inline docs/playbook.md:44]: Run this command to fetch default creds to the vm. This creds are copied to x place

**Review by vadlakondaswetha — CHANGES_REQUESTED** [review]

File: docs/playbook.md [hunk 4]
```diff
@@ -36,6 +36,18 @@ Command to enable the flag:
...
+Run "gcloud auth application-default login" command to fetch default creds
```
Comment (vadlakondaswetha) [inline docs/playbook.md:45]: you already mentioned
resolve this issue by fetching creds with below command
why do you need to say run x command again. just give the command.

**Review by vadlakondaswetha — APPROVED** [review]

File: docs/playbook.md (hunk 4 above)
Comment (raj-prince) [reply docs/playbook.md:44]: That was a mistake, forgot to delete the above line. Thanks!

**Pull request merged**